src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,46 @@
+package duke.ui;
+
+import java.util.List;
+import java.util.Scanner;
+import duke.task.TaskList;
+import duke.task.Task;
+
+public class Ui {
+    private Scanner sc;
+    private final String helloMessage = "Hello! I'm Clara! :D How may I help you? :)";
```

Comment:
Final constant names should be capitalised, eg. `HELLO_MESSAGE`, `BYE_MESSAGE`, `ERROR_HEADER`.

Reply:
Thanks, will update it accordingly 👍